locales/en.json

```
@@ -190,7 +190,19 @@
             "invalid_year": "The card's expiration year is invalid.",
             "invalid_month": "The card's expiration month is invalid.",
             "invalid_cvv": "The card's CVV is invalid.",
-            "expired": "The card's expiration date is passed"
+            "expired": "The card's expiration date is passed.",
+            "declined": "The card has been declined. Please call your card issuer for more information.",
+            "address_invalid": "The card’s address is incorrect.",
+            "insufficient_funds": "The account did not have sufficient funds to cover the transaction amount.",
+            "square": {
+                "card_not_supported": "The card is not supported in the geographic region."
+            },
+            "paypal": {
+                "currency_not_supported": "The currency is not currently supported by Paypal."

```

The right way to write PayPal is with two capital Ps.

```suggestion
                "currency_not_supported": "The currency is not currently supported by PayPal."

```